code/modules/vending/wardrobes.dm

```diff
@@ -0,0 +1,385 @@
+					/obj/item/clothing/shoes/jackboots = 3,
+					/obj/item/clothing/head/beret/sec = 3,
+					/obj/item/clothing/head/soft/sec = 3,
+					/obj/item/clothing/mask/bandana/red = 3)
+	refill_canister = /obj/item/vending_refill/wardrobe/sec_wardrobe
+
+/obj/item/vending_refill/wardrobe/sec_wardrobe
+	machine_name = "SecDrobe"
+	charges = list(8, 0, 0)
+	init_charges = list(8, 0, 0)
+
+/obj/machinery/vending/wardrobe/medi_wardrobe
+	name = "\improper MediDrobe"
+	desc = "A vending machine rumoured to be capable in dispensing clothing for medical personnel."
+	product_ads = "Make those blood stains look fashionable!!"
+	req_access_txt = "5"
+	vend_reply = "Thank you for using the MediDrobe!"
+	products = list(/obj/item/clothing/accessory/pocketprotector = 1,
+					/obj/item/storage/backpack/duffelbag/med = 1,
+					/obj/item/storage/backpack/medic = 1,
+					/obj/item/storage/backpack/satchel/med = 1,
+					/obj/item/clothing/suit/hooded/wintercoat/medical = 1,
+					/obj/item/clothing/under/rank/nursesuit = 1,
+					/obj/item/clothing/head/nursehat = 1,
+					/obj/item/clothing/under/rank/medical/blue = 1,
+					/obj/item/clothing/under/rank/medical/green = 1,
+					/obj/item/clothing/under/rank/medical/purple = 1,
+					/obj/item/clothing/under/rank/medical = 3,
+					/obj/item/clothing/suit/toggle/labcoat = 3,
+					/obj/item/clothing/suit/toggle/labcoat/emt = 3,
+					/obj/item/clothing/shoes/sneakers/white = 3,
+					/obj/item/clothing/head/soft/emt = 3)
+	refill_canister = /obj/item/vending_refill/wardrobe/medi_wardrobe
+
+/obj/item/vending_refill/wardrobe/medi_wardrobe
+	machine_name = "MediDrobe"
+	charges = list(10, 0, 0)
+	init_charges = list(10, 0, 0)
+
+/obj/machinery/vending/wardrobe/engi_wardrobe
+	name = "EngiDrobe"
+	desc = "A vending machine reknowned for vending industrial grade clothing."
+	product_ads = "Gauranteed to protect your feet from industrial accidents!; Afraid of radiation? Then wear yellow!"
+	req_access_txt = "11"
+	vend_reply = "Thank you for using the EngiDrobe!"
+	products = list(/obj/item/clothing/accessory/pocketprotector = 1,
+					/obj/item/storage/backpack/duffelbag/engineering = 1,
+					/obj/item/storage/backpack/industrial = 1,
+					/obj/item/storage/backpack/satchel/eng = 1,
+					/obj/item/clothing/suit/hooded/wintercoat/engineering = 1,
+					/obj/item/clothing/under/rank/engineer = 3,
+					/obj/item/clothing/suit/hazardvest = 3,
+					/obj/item/clothing/shoes/workboots = 3,
+					/obj/item/clothing/head/hardhat = 3)
+	refill_canister = /obj/item/vending_refill/wardrobe/engi_wardrobe
+
+/obj/item/vending_refill/wardrobe/engi_wardrobe
+	machine_name = "EngiDrobe"
+	charges = list(7, 0, 0)
+	init_charges = list(7, 0, 0)
+
+/obj/machinery/vending/wardrobe/atmos_wardrobe
+	name = "AtmosDrobe"
+	desc = "This relatively unknown vending machine delivers clothing for Atmospherics Technicions, an equally unknown job."
+	product_ads = "Get your inflammible clothing right here!!!"

```

Comment:
inflammable

Reply:
Hope you get the joke with inflammable though.

Reply:
The English language really pisses me off sometimes; this is one of those times.